From 0f7521d8ec036f37b476eae20e995934962940f2 Mon Sep 17 00:00:00 2001
From: Adrian Bradianu <abradianu@gmail.com>
Date: Tue, 26 Jul 2016 23:09:04 +0300
Subject: [PATCH 2/2] sunxi: Set CPUX-VSET GPIO to low

Enable CPUX_VSET_GPIO_LOW and set SYS_CLK_FREQ to 480MHz in order
to decrease CPU temperature.

Signed-off-by: Adrian Bradianu <abradianu@gmail.com>
---
 configs/orangepi_one_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/orangepi_one_defconfig b/configs/orangepi_one_defconfig
index de6e9c8..a4b3de4 100644
--- a/configs/orangepi_one_defconfig
+++ b/configs/orangepi_one_defconfig
@@ -23,3 +23,5 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_FAT=y
 CONFIG_CMD_FS_GENERIC=y
 CONFIG_USB_EHCI_HCD=y
+CONFIG_SYS_CLK_FREQ=408000000
+CONFIG_SYS_EXTRA_OPTIONS="CPUX_VSET_GPIO_LOW"
-- 
1.9.1

